### ISSUE TITLE
Missing cross references for calls to ImportedAddressValue

### ISSUE THREAD

**Issue by loverics** [Vector35/binaryninja-api#1371]
The following code does not work for mipsel binaries. Specifically  `xref.function.get_low_level_il_at(xref.address).mlil` returns None. I've attached the mipsel binary that will reproduce the bug.

```
memcpy_symbol = bv.get_symbol_by_raw_name("memcpy@GOT")

print(f'{memcpy_symbol}\n')

xrefs = bv.get_code_refs(memcpy_symbol.address)

print(f'Found {len(xrefs)} xrefs to memcpy\n')

for xref in xrefs:
    mlil = xref.function.get_low_level_il_at(xref.address).mlil
    
    if mlil.operation == binja.MediumLevelILOperation.MLIL_CALL:
        if len(mlil.operands[2]) == 3:
            print(f'{xref.function.name}: xref @ {hex(xref.address)} operands[2]: {mlil.operands[2]}')
```

[memcpy_test_linux_mipsel.zip](https://github.com/Vector35/binaryninja-api/files/3283824/memcpy_test_linux_mipsel.zip)

Here's the output from the mipsel binary:

```
<SymbolType.ImportAddressSymbol: "memcpy@GOT" @ 0x410c74>

Found 5 xrefs to memcpy

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-498-578cec41a998> in <module>
     10     mlil = xref.function.get_low_level_il_at(xref.address).mlil
     11 
---> 12     if mlil.operation == binja.MediumLevelILOperation.MLIL_CALL:
     13         if len(mlil.operands[2]) == 3:
     14             print(f'{xref.function.name}: xref @ {hex(xref.address)} operands[2]: {mlil.operands[2]}')

AttributeError: 'NoneType' object has no attribute 'operation'
```
I tried to work around this bug by using `xref.function.get_low_level_il_at(xref.address).mapped_medium_level_il` however the operands for the mlil return something similar to llil operands. For the full set of mlil operands I expected to see something like `[<il: $v0_1 = memcpy($a0, $a1, $a2)>]` and instead this was returned `[<var >, <il: memcpy>]`.

The above code works with arm7, mips and ppc32 versions of the same binary. I've attached the arm7 binary that works. For the arm7 binary I changed `memcpy_symbol = bv.get_symbol_by_raw_name("memcpy@GOT")` to `memcpy_symbol = bv.get_symbol_by_raw_name("memcpy")`

[memcpy_test_linux_arm7.zip](https://github.com/Vector35/binaryninja-api/files/3283826/memcpy_test_linux_arm7.zip)

I generated the arm7, mips, mipsel binaries with dockcross https://github.com/dockcross/dockcross and I used the following gcc command line:

`gcc O0 -fno-builtin -g -D_FORTIFY_SOURCE=0 -fno-stack-protector test/C/memcpy_test.c -o memcpy_test_linux_mipsel`

Kind regards,
Shawn

**Comment by loverics** [comment]
I'm using the API from commit 609de737864e3768ff35830268c733e6b4ee7ccd

**Comment by plafosse** [comment]
Unfortunately this is a know limitation of the current API. We recommend iterating the MLIL instead of walking the LLIL and trying to map to MLIL.

**Comment by plafosse** [comment]
Changing the title to reflect the underlying issue.

**Comment by bpotchik** [comment]
We can improve instruction mappings, however in some cases there simply is not valid mapping from lower level ILs to higher level ILs. It's recommended to write your code with the expectation that a mapping may not exist.

For this particular issue the underlying bug seems to be missing cross references for calls to ImportedAddressValue. This is especially noticeable on MIPS. This issue is now fixed in build `2.1.2449-dev`

**Comment by bpotchik** [comment]
Improvements to instruction mappings between ILs can be followed in #2023
